docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -3493,6 +3499,11 @@ the NOT NULL column constraints that exist for the source table with different c
 directs {project-name} SQL to not use divisioning from _source-table_. If this clause is omitted, then 
 the _table_ will have the same divisioning as the _source-table_.
 
+*** `WITHOUT LOB COLUMNS`
++
+directs {project-name} SQL to not use LOB columns from the _source-table_. If this clause is omitted, then 

```

Comment:
Suggested wordsmith: "directs {project-name} SQL to not include LOB columns..."

Reply:
Thank you, Dave. I've incorporated your comment.